lib/src/common/js_proxy.dart

```diff
@@ -145,11 +145,32 @@ dynamic dartValue(jsValue) {
       }
       _addDartInstance(jsValue, dartMap);
       return dartMap;
+    } else if (jsValue is JsFunction) {
+      return _dartType(jsValue);
```

Comment:
In Polymer.js land you can specify the type of a property. So then it comes back with the function for the constructor when you override the deserialize.